packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -114,33 +137,275 @@ describe('Collection aggregations tab', function () {
+    const collationInput = await browser.$(Selectors.AggregationCollationInput);
+    await collationInput.waitForDisplayed();
+    await collationInput.setValue('{ locale: "af" }');
+
+    // select $match
+    await browser.focusStageOperator(0);
+    await browser.selectStageOperator(0, '$match');
+    // check that it included the comment by default
+    const contentElement0 = await browser.$(Selectors.stageContent(0));
+
+    // It starts out empty
+    await browser.waitUntil(async () => {
+      const text = await contentElement0.getText();
+      return text !== '';
+    });
+
+    expect(await contentElement0.getText()).to.equal(`/**
+ * query: The query in MQL.
+ */
+{
+  query
+}`);
+
+    //change $match
+    await browser.setAceValue(Selectors.stageEditor(0), '{ i: { $gt: 5 } }');
+
+    // TODO: click collapse and then expand again
+
+    // open settings
+    await browser.clickVisible(Selectors.AggregationSettingsButton);
+
+    // turn off comment mode
+    await browser.clickVisible(Selectors.AggregationCommentModeCheckbox);
+
+    // set number of preview documents to 100
+    const sampleSizeElement = await browser.$(
+      Selectors.AggregationSampleSizeInput
+    );
+    await sampleSizeElement.setValue('100');
+
+    // apply settings
+    await browser.clickVisible(Selectors.AggregationSettingsApplyButton);
+
+    // add a $project
+    await browser.clickVisible(Selectors.AddStageButton);
+    await browser.focusStageOperator(1);
+    await browser.selectStageOperator(1, '$project');
+
+    // delete it
+    await browser.clickVisible(Selectors.stageDelete(1));
+
+    // add a $project
+    await browser.clickVisible(Selectors.AddStageButton);
+    await browser.focusStageOperator(1);
+    await browser.selectStageOperator(1, '$project');
+
+    // check that it has no comment
+    const contentElement1 = await browser.$(Selectors.stageContent(1));
+
+    // starts empty
+    await browser.waitUntil(async () => {
+      const text = await contentElement1.getText();
+      return text !== '';
+    });
+
+    expect(await contentElement1.getText()).to.equal(`{
+  specification(s)
+}`);
+    await browser.setAceValue(Selectors.stageEditor(1), '{ _id: 0 }');
+
+    // disable it
+    await browser.clickVisible(Selectors.stageToggle(1));
+
+    // export to language
+    await browser.clickVisible(Selectors.ExportAggregationToLanguage);
+    const text = await browser.exportToLanguage('Ruby');
+    expect(text).to.equal(`[
+  {
+    '$match' => {
+      'i' => {
+        '$gt' => 5
+      }
+    }
+  }
+]`);
+
+    // check that the preview is using 100 docs
+    await browser.waitUntil(async function () {
+      const textElement = await browser.$(
+        Selectors.stagePreviewToolbarTooltip(0)
+      );
+      const text = await textElement.getText();
+      return text === '(Sample of 100 documents)';
+    });
+
+    // save as a view
+    // TODO: This is currently broken, so will have to test at a later stage
```

Comment:
We can do this now or after this PR. Actually creating a view works, it is the browsing to the view that was broken.